docs/compendium/substitute.md

```diff
@@ -128,6 +131,54 @@ Result will be a list of compendia ids that have been substituted out of a choos
 }
 ```
 
-### URL parameters for substititions
+## List related substituted Compendia
+
+Filter by base and/or overlay compendium id
+
+- `GET /api/v1/substitution/?base=compendium_id&overlay=compendium_id`
```

Comment:
We don't have the trailing slash in other examples, but we do use either concrete values or some formatting of placeholders (see [here](http://o2r.info/o2r-web-api/compendium/view/#view-single-compendium): suggest to use `../substitution?base=$BASE_ID...` in the template and a concrete id in the examples, ``../substitution?base=oMM15` 

Please also add examples for all cases (only base, only overlay, both) with concrete values, similar to http://o2r.info/o2r-web-api/compendium/view/#list-compendia I know this seems unnecessary, but the more explicit we are now, the better we know later what we originally intended. Thanks!